basic_network.go

```diff
@@ -302,6 +302,15 @@ func (n *BasicVideoNetwork) GetMasterPlaylist(p string, manifestID string) (chan
 
 		return returnC, nil
 	}
+	nid, err := extractNodeID(manifestID)
+	if err != nil {
+		glog.Errorf("Error extracting NodeID from: %v", manifestID)
+		return nil, ErrGetMasterPlaylist
+	}
+	if n.GetNodeID() == peer.IDHexEncode(nid) {
+		glog.Errorf("Node ID from manifest:%v is the same as the current node ID: %v", peer.IDHexEncode(nid), n.GetNodeID())
```

Comment:
Was this a problem? The node being its own transcoder and sending itself a response?

Reply:
Yeah I think the tests had a bug.  Should be fixed.